app/views/albums/_form.html.erb

```diff
@@ -23,6 +23,13 @@
   </div>
 
   <% if page_type == :new %>
+    <div class="field">
+      <%= form.label "Album köre", class: "label" %>
+      <div class="select">
+        <%= form.select :circle_id, circles.collect {|c| [c.name, c.id]}, {}, class: "" %>
```

Comment:
I think we can left out the empt options hash and class attribute.

```suggestion
        <%= form.select :circle_id, circles.collect {|c| [c.name, c.id]}%>
```

Reply:
Yeah I just left it in during development just in case I wanted to add some css classes to it (I always forget, that there should be an empty object for options, that's why its left there), gonna mend it soon